Sources/BraintreeThreeDSecure/BTThreeDSecureClient.swift

```diff
@@ -103,7 +103,7 @@ import BraintreeCore
             return
         }
 
-        guard request.nonce != nil else {
+        if request.nonce.isEmpty {
             notifyFailure(
                 with: BTThreeDSecureError.configuration("BTThreeDSecureRequest nonce can not be nil."),
```

Comment:
Same here, can also update to something like "BTThreeDSecureRequest nonce can not be an empty string"